sftpplus-docker-setup.sh

```diff
@@ -0,0 +1,58 @@
+# To be used with the SFTPPlus Dockerfile.
```

Comment:
this is ok... and is on my taste.... but for the purpose of the example and learning not sure if it is not better to keep thing simple and have separate files for each OK.... I don't know  :)

Reply:
It think things are more simple with one common script. Documentation is simpler and the structure of the repo is simplified. True, at the cost of a slightly more complex setup script. But even this comes with benefit of highlighting OS-dependent commands with the `case` constructs.